crates/starlight/src/bytecompiler.rs

```diff
@@ -2272,6 +2295,23 @@ fn is_builtin_call(e: &Expr, builtin_compilation: bool) -> bool {
     false
 }
 impl ByteCompiler {
+    pub fn handle_codegen_plugin_call(&mut self, call: &CallExpr) -> Result<(), JsValue> {
+        let plugin_name = if let ExprOrSuper::Expr(expr) = &call.callee {
+            if let Expr::Ident(x) = &**expr {
+                let str = &*x.sym;
+                str
+            } else {
+                unreachable!();
```

Comment:
This code should not panic but return an error. Maybe syntax error? Codegen plugins is public to users so users should get or throw errors from these plugins too.